engine/src/flutter/display_list/dl_text.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_DISPLAY_LIST_DL_TEXT_H_
+#define FLUTTER_DISPLAY_LIST_DL_TEXT_H_
+
+#include "flutter/display_list/geometry/dl_geometry_types.h"
+#include "third_party/skia/include/core/SkRefCnt.h"
+
+namespace impeller {
+class TextFrame;
+}
+
+class SkTextBlob;
+
+namespace flutter {
+class DlText {
+ public:
+  virtual DlRect getBounds() const = 0;
```

Comment:
I know there is a mixed style in DL objects, but I think we are moving towards capitalized getters like GetBounds, GetTextFrame, etc. I wasn't doing that until I started integrating with Impeller, but I think it is better matched to the C++ style guidelines so I'd like to move in that direction.

Technically you can have a strict getter that is lower case, but I don't think either of getTextFoo() are "strict getters" since they are only defined for specific subclasses...